config/yamllint.yaml

```diff
@@ -0,0 +1,37 @@
+---
+
+yaml-files:
+  - '*.yaml'
+  - '*.yml'
+
+rules:
+  braces: enable
+  brackets:
+    min-spaces-inside: 1
+    max-spaces-inside: 3
+  colons: enable
+  commas: enable
+  comments: enable
+  comments-indentation: enable
+  # We have no use for yaml directives, and we do not have single yaml files
+  # with multiple documents in them, so we do not need doc start or end.
+  document-start: disable
+  document-end: disable
+
+  empty-lines: enable
+  empty-values: enable
+  hyphens: enable
+  indentation: enable
+  key-duplicates: enable
+  # alphabetical keys are not always the best way to organize them
+  key-ordering: disable
+  line-length:
+    max: 100
+  new-line-at-end-of-file: enable
+  new-lines: enable
+  octal-values: enable
+  # we do not need such strict format
+  quoted-strings: disable
```

Comment:
I think this is probably overkill, but if we decide to activate this rule, let's make a separate issue:
```
➜  checkstyle git:(issue-12435) ✗ git diff | cat
diff --git a/config/yamllint.yaml b/config/yamllint.yaml
index 779d79290..9f1d4e645 100644
--- a/config/yamllint.yaml
+++ b/config/yamllint.yaml
@@ -31,7 +31,7 @@ rules:
   new-lines: enable
   octal-values: enable
   # we do not need such strict format
-  quoted-strings: disable
+  quoted-strings: enable
   trailing-spaces: enable
   truthy:
     allowed-values: [ 'true', 'false', 'on' ]
➜  checkstyle git:(issue-12435) ✗ yamllint -f parsable -c config/yamllint.yaml .  | wc -l
605

```

Reply:
Example of what its asking for?

Reply:
`quoted-strings: "enable"`

Reply:
To me, this just makes some simple yaml more complicated than it needs to be. There are some cases where I use quotes, but I do not think it should be required.

Reply:
We might coy to this fanatism at next phase